MdePkg/Include/IndustryStandard/Usb.h

```diff
@@ -131,6 +133,21 @@ typedef struct {
   UINT8     MaxPower;
 } USB_CONFIG_DESCRIPTOR;
 
+///
+/// Standard Interface Association Descriptor
+/// USB Engineering Change Note "Interface Association Descriptors" Revision 2.0, Page 4

```

Comment:
Is this only in an ECN and not in a full spec release?  What is the link to the ECN?

Reply:
It is included in the spec starting from USB3.0. The ECN is https://www.usb.org/sites/default/files/iadclasscode_r10.pdf